docs/architecture/flow/README.md

```diff
@@ -0,0 +1,17 @@
+# Flow diagrams
+
+The following diagrams shows Application registration and Runtime creation flows.
+
+## Application registration flow
```

Comment:
Maybe we can reuse most of the existing steps of Application registration diagram? This will show that we did not break anything regarding the compatibility.

Reply:
The diagram has been updated.